msu/hooks/skills/skill.nut

```diff
@@ -310,6 +311,38 @@
 		return ret;
 	}
 
+	local removeSelf = o.removeSelf;
+	o.removeSelf = function()
+	{
+		if (this.isStacking())
+		{
+			removeSelf();
+		}
+		else
+		{
+			this.m.MSU_AddedStack--;
+			if (this.m.MSU_AddedStack <= 0) removeSelf();
```

Comment:
I've noticed you don't do this in other places too, and I'd wanna review the codebase a bit for it, but part of the charm of in/decrement operators is that they never have to be their own line. Therefore this can be replaced with

```squirrel
if (--this.m.MSU_AddedStack <= 0) removeSelf();
```

Reply:
Done